test/paper-button-behavior.html

```diff
@@ -95,13 +69,12 @@
             }
           });
         });
-
-        test('receives focused', function() {
-          MockInteractions.focus(button);
-          assert.equal(button._elevation, 2);
-        })
       });
 
+      test('receives focused', function() {
+        MockInteractions.focus(button);
+        assert.equal(button._elevation, 3);
```

Comment:
Does this work? Do you need a `keyUp` or down to set `receivedFocusFromKeyboard` correctly?